projects/jsf-launcher/src/app/schemaV2.json

```diff
@@ -969,6 +969,39 @@
           }
         }
       }
+    },
+    "templateTest": {
```

Comment:
There's a dataV2.json file that is used when AppComponent is set to edit mode, can you check if we should add anything for the template's target values? This will simulate what it looks like when loading previously set data.